README.md

```diff
@@ -100,23 +100,27 @@ Binaries, installers, and source tarballs are available at
 <https://nodejs.org/en/download/>.
 
 #### Current and LTS Releases
-**Current** and **LTS** releases are available at
-<https://nodejs.org/download/release/>, listed under their version strings.
+<https://nodejs.org/download/release/>
+
 The [latest](https://nodejs.org/download/release/latest/) directory is an
-alias for the latest Current release. The latest LTS release from an LTS
-line is available in the form: latest-_codename_. For example:
-<https://nodejs.org/download/release/latest-argon>.
+alias for the latest Current release. The latest-_codename_ directory is an
+alias for the latest release from an LTS line. For example,
+<https://nodejs.org/download/release/latest-carbon> is the latest Carbon
+(Node.js version 8) release.
 
 #### Nightly Releases
-**Nightly** builds are available at
-<https://nodejs.org/download/nightly/>, listed under their version
-string which includes their date (in UTC time) and the commit SHA at
-the HEAD of the release.
+<https://nodejs.org/download/nightly/>
+
+Release names are appeded a version string that includes the date (in UTC time)
+and the 10 first characters of the commit SHA at the HEAD of the release.
```

Comment:
Typo `appeded` -> `appended with`.

Reply:
On reflection I think I prefer the original text. The reworded text is a little more awkward to read and adds more technical specifics for no added value.

Reply:
Ack.